WebhookNotificationActivity/index.ts

```diff
@@ -44,14 +53,26 @@ const notificationModel = new NotificationModel(
 // Whether we're in a production environment
 const isProduction = process.env.NODE_ENV === "production";
 
+// Webhook must be an https endpoint
+const abortableFetch = AbortableFetch(agent.getHttpsFetch(process.env));
```

Comment:
why `getHttpsFetch` needs to have access to all the environment? we should minimize the data required, can we just pass the relevant env vars?

Reply:
the whole point of this utility method is to abstract away the processing of options from the environment.

if we make it accepts only the required properties we end up reply (cut&paste) all this logic wherever you have to get a new agent: https://github.com/pagopa/io-ts-commons/blob/master/src/agent.ts#L10

which make the whole agent module useless

Reply:
ok I see 👍